gradle.properties

```diff
@@ -114,8 +114,8 @@ artifactRedirecting.androidx.compose.material3.adaptive.version=1.1.0-beta01
 artifactRedirecting.androidx.collection.version=1.5.0-beta01
```

Comment:
> Highlights - Multiple Platforms

1. Do we need it in "Highlights" (major Features) or in "Features" (minor features)?
2. "Multiple Platforms" is reserved for Compose by initial idea (aka "Compose - Multiple Platforms". For Navigation we have "Navigation". It is not obvious, but other schemes need discussion.

Reply:
> Do we need it in "Highlights" (major Features) or in "Features" (minor features)?

We we need to describe some of the features of this features, add them ass sub items. If we don't need, we usually don't add Release Notes. There will be a link to the Jetpack Navigation release notes

Reply:
Added a link to the corresponding androidx version

Reply:
In case of a simple version update, we need to use `N/A` in this PR. The link will be in "Dependencies" block

Reply:
> Breaking Changes - Multiple Platforms

It should be

> Breaking Changes - Navigation

Reply:
> Multiplatform navigation was migrated from a temporary internal core-bundle module to the androidx SavedState

Does it affect public navigation API?

Reply:
> "Multiple Platforms" is reserved for Compose by initial idea

Okay, I though about it in https://github.com/JetBrains/compose-multiplatform/pull/5240/files, and it seems we need a category for something that doesn't fall into existing ones.

Because of this, we could use "Multiple Platforms". Currently only Compose fall into this category. 

For Navigation though we have a separate category, so better to use "Navigation" instead of "Multiple Platforms"